endpoints/feedback.py

```
@@ -0,0 +1,140 @@
+    if not feedback_enabled:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="Forbidden: Feedback is disabled",
+        )
+
+
+@router.post("", responses=feedback_response)
+def feedback_endpoint_handler(
+    feedback_request: FeedbackRequest,
+    auth: Annotated[AuthTuple, Depends(auth_dependency)],
+    _ensure_feedback_enabled: Any = Depends(assert_feedback_enabled),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)